### PR TITLE
Add tracing library metadata to local root span

### PR DESCRIPTION
**Pull request by owais**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,3 +77,15 @@ idea about the versions and the set-up take a look at our [CI config](https://gi
 
 The best way to run the entire test suite is using the [CircleCI CLI](https://circleci.com/docs/2.0/local-jobs/). Simply run `circleci build`
 in the repository root. Note that you might have to increase the resources dedicated to Docker to around 4GB.
+
+
+
+## Contributing
```
Comment (rmfitzpatrick) [inline README.md:83]: Should this go in CONTRIBUTING.md?

File: tracing/tracing.go [hunk 2]
```diff
@@ -103,6 +107,13 @@ func WithEndpointURL(url string) StartOption {
 	}
 }
 
+// WithoutLibraryTags configures the URL to send traces to
```
Comment (rmfitzpatrick) [inline tracing/tracing.go:110]: c/p issue

File: ddtrace/ext/tags.go [hunk 3]
```diff
@@ -99,4 +99,10 @@ const (
 
 	// Stack is a stack trace
 	Stack = "stack"
+
+	// SFXTracingLibrary specifies the SignalFx tracing library that generated the span.
+	SFXTracingLibrary = "sfx.tracing.library"
```
Comment (rmfitzpatrick) [inline ddtrace/ext/tags.go:104]: Other libraries have been using full `signalfx` prefix.

**Comment by rmfitzpatrick** [comment]
Would you be opposed to adopting https://github.com/signalfx/signalfx-go-tracing/pull/31/files#diff-f88a13755a4b2ad7cfcd12f7defdd3a2R26 in this pass?

**Comment by owais** [comment]
@rmfitzpatrick PTAL

**Pull request merged**